### ISSUE TITLE
Update wait for cron-job

### ISSUE THREAD

**Issue by ShinjiLE** [nextcloud/server#10510]
### Steps to reproduce
1. download 14.0.0 Beta 2
2. run updater.phar
3. occ upgrade

### Expected behaviour
Upgrade should going on .

### Actual behaviour
Updater is waiting endless for the cron-job

### Server configuration

**Operating system**: debian buster/sid

**Web server:** Apache/2.4.34 (Debian)

**Database:** mariadb  Ver 15.1 Distrib 10.1.34-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

**PHP version:** PHP 7.2.8-2

**Nextcloud version:** version: 14.0.0.14

**Updated from an older Nextcloud/ownCloud or fresh install:** updated from 14.0 beta 1

**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>
 - accessibility: 1.0.1
  - activity: 2.7.0
  - admin_audit: 1.4.0
  - audioplayer: 2.3.1
  - bookmarks: 0.11.0
  - bookmarks_fulltextsearch: 0.8.0
  - bruteforcesettings: 1.1.0
  - calendar: 1.6.1
  - cloud_federation_api: 0.0.1   
  - comments: 1.4.0
  - contacts: 2.1.5
  - dashboard: 5.0.0
  - dav: 1.6.0
  - federatedfilesharing: 1.4.0   
  - files: 1.9.0
  - files_automatedtagging: 1.4.0 
  - files_external: 1.5.0
  - files_fulltextsearch: 0.8.2   
  - files_fulltextsearch_tesseract: 0.1.0
  - files_markdown: 2.0.4
  - files_pdfviewer: 1.3.2
  - files_rightclick: 0.8.4
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - firstrunwizard: 2.3.0
- fulltextsearch: 0.8.1
  - fulltextsearch_elasticsearch: 0.8.0
  - gallery: 18.1.0
  - groupfolders: 1.2.2
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - nextcloud_announcements: 1.3.0
  - notes: 2.3.2
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - passman: 2.1.5
  - password_policy: 1.4.0
  - previewgenerator: 1.1.0
  - provisioning_api: 1.4.0
  - richdocuments: 2.0.10
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - support: 1.0.0
  - survey_client: 1.2.0
  - systemtags: 1.4.0
  - twofactor_backupcodes: 1.3.0  
  - updatenotification: 1.4.1
  - workflowengine: 1.4.0
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.mybase.selfhost.de"
        ],  
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "tempdirectory": "\/var\/tmp\/nextcloudtemp",
        "overwrite.cli.url": "https:\/\/nextcloud.mybase.selfhost.de",
        "knowledgebaseenabled": true,
        "dbtype": "mysql",
        "version": "14.0.0.13",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
"dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "updater.release.channel": "beta",
        "appstore.experimental.enabled": true,
        "maintenance": true,
        "theme": "",
        "loglevel": 0,
        "debug": false,
        "htaccess.RewriteBase": "\/",
        "enable_previews": true,  
        "preview_max_x": 2048,
        "preview_max_y": 2048,
        "preview_max_filesize_image": 300,
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "preview_office_cl_parameters": " --headless --nologo --nofirststartwizard --invisible --norestore --convert-to pdf --outdir ",
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",   
            "OC\\Preview\\JPEG",  
            "OC\\Preview\\GIF",   
            "OC\\Preview\\BMP",   
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\MP3",   
            "OC\\Preview\\TXT",   
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\TIFF",  
            "OC\\Preview\\SVG",   
            "OC\\Preview\\Postscript",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\PDF",  
"OC\\Preview\\Font",  
            "OC\\Preview\\Epub",  
            "OC\\Preview\\StarOffice"
        ],
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",  
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "log_type": "file",
        "syslog_tag": "Nextcloud",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": "true",
        "filelocking.ttl": 3600,  
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 0,
            "dbindex": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "data-fingerprint": "9a4e1a080bac1d929a6686fe4d9c9276",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls", 
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
</details>

**Are you using external storage, if yes which one:** local

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:**

**Operating system:**

### Logs
<details>
<summary>occ update output</summary>
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Set log level to debug
Waiting for cron to finish (checks again in 5 seconds) …
Waiting for cron to finish (checks again in 5 seconds) …
</details>

**Comment by ShinjiLE** [comment]
Problem solved , but no solution . And i don't know what really happen before. 
I have delete the complete Nextcloud installation folder except the configuration and replaced it with the content of the nextcloud-14.0.0beta2.zip . Now all is ok .

Should this ticket closed ?

**Comment by newsyrom** [comment]
i had the same problem yesterday and after digging and digging i found out that the has something to do with the database. so i was looking ferther in the database and the connection and the users. at the end i found something weard . in the OC _appconfig table there are a few values . the one that fixed my problem was the installedat and the last cron and the lastupdateresult and the lastupdatedat. one of this solved my problem because i just made them blank . now every thing is working as it suppose to be.
but during the digging i deleted a folder from the data and now he is asking for the files and i do not know from where to get them.

""No cache entry found for /appdata_ociqwo4jvxq3/css/icons/icons-vars.css (storage: local::/home/user/web/domain.com/public_html/nextcloud/data/, internalPath: appdata_ociqwo4jvxq3/css/icons/icons-vars.css)

can anybody lead me to solve it?

**Comment by MorrisJobke** [comment]
Duplicate of #9992

**Comment by MorrisJobke** [comment]
Fix was in #10010
